hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/OzoneBucket.java

```diff
@@ -155,16 +161,19 @@ public OzoneBucket(Configuration conf, ClientProtocol proxy,
    * @param storageType StorageType of the bucket.
    * @param versioning versioning status of the bucket.
    * @param creationTime creation time of the bucket.
+   * @param modificationTime modification time of the bucket.
    */
   @SuppressWarnings("parameternumber")
   public OzoneBucket(Configuration conf, ClientProtocol proxy,
```

Comment:
Do we consider the backward compatibility here when changing the public client APIs? This will be important moving forward from Beta to GA.

Reply:
Thanks @xiaoyuyao for the review.

Do you mean that we should have a new API such as 
```  
public OzoneBucket(Configuration conf, ClientProtocol proxy, 
    String volumeName, String bucketName, StorageType storageType,
    Boolean versioning, long creationTime, long modificationTime,
    Map<String, String> metadata) {
  this.modificationTime = modificationTime;
  OzoneBucket(conf, proxy, volumeName, bucketName, storageType, versioning, 
    creationTime, metadata); // existed API.
}
```
here ?

Reply:
Thanks @xiaoyuyao for the idea.
Updated.